examples/switch/switch.go

```diff
@@ -60,4 +60,21 @@ func main() {
 	whatAmI(true)
 	whatAmI(1)
 	whatAmI("hey")
+
+	// Every case includes a break by default.
+	// You can use the keyword 'fallthrough' to override this
+	// behaviour. In this example, we print 'FooBar' if num
+	// is even and 'Bar'  if num is odd.
```

Comment:
I'm not sure about this because "break" presupposes C background which many readers may not possess. For example the Go Spec when describing `fallthrough` doesn't do this, but explains it from first principles.

IMHO this is too much of an arcana for gobyexample -- `fallthrough` is pretty rare in practice

Reply:
I agree with your first point.

But I had a need to use `fallthrough`.
Inspite of completing the Go Tour, I had to Google this up as it was not covered.
Ended  up finding this information in [Golang Wiki](https://github.com/golang/go/wiki/Switch#fall-through).

So, I felt this extra information might be helpful to at least a few.